packages/client/.eslintignore

```diff
@@ -0,0 +1,3 @@
+build
+node_modules
+tests
```

Comment:
😅 , tbh I pretty much copy pasted the configuration from the Core / Init

Reply:
:/ so there maybe more configuration that reuiqred?

Reply:
I was thinking, the deploy alpha might fail as it doesn't build the client first also. (same as core)
For which we might want to create a command like "build essential packages" to prevent that from happening again.